### ISSUE TITLE
Deadlight (211400)

### ISSUE THREAD

**Issue by leillo1975** [ValveSoftware/Proton#910]
The game seems to work perfect in my system:

https://gist.github.com/leillo1975/93f3b9ee02cb2b47e8de00dfe40f6957

**Comment by NoXPhasma** [comment]
I can not confirm that this game is ready to be whitelisted, it has weird graphical glitches for me: https://i.imgur.com/mdEnFQX.jpg

https://gist.github.com/NoXPhasma/17aaaab0d5841d6a92dac4d184a9592f

**Comment by leillo1975** [comment]
I don't have this problems.

**Comment by NoXPhasma** [comment]
Yes, I guess so, otherwise you wouldn't have created this issue. But if the game doesn't work properly for the majority, it shouldn't be whitelisted. Do you agree?

**Comment by jalabb** [comment]
I'm having the same issue (i5 6600K, 1060 6GB)
https://imgur.com/a/KjiOl86
https://gist.github.com/poke86/ec7cdb9003481d0452dac59da07e37f3

**Comment by corvusd** [comment]
@leillo1975 please report exact configuration system specs and proton values?

I open bug in winehq for wine-staging version, right now send the trace log of the game for this error request by user/devel.

https://bugs.winehq.org/show_bug.cgi?id=45978

My SF specs env:
Wine-Staging 3.18 ppa oficial / Proton 3.7-6

My HW specs:
AMD Phenom(tm) II X6 1055T Processor × 6
8GB RAM DDR3-1333
SSD Sata3 KINGSTON 120Gb / Seagate Sata3 HDD 1TB
Nvidia GeForce GTX 1060 OC 3072MB VRAM DDR5
Ubuntu 16.04.5 LTS (Last updated) / Unity 7
Driver Nvidia: 396.54.09 official graphics ppa
Gamepad Xbox360 Wired USB
Steam Gamepad
Webcam Logitech QuickCam E 3500

**Comment by corvusd** [comment]
In winehq commented that maybe problem of driver vendor. 
https://bugs.winehq.org/show_bug.cgi?id=45978#c8

We can try to report to nvidia forums: 
https://devtalk.nvidia.com/default/board/98/linux/

Now it seems that nvidia is considering wine / proton to fix these problems. If someone opens a thread, please post the link here, so that we can all concentrate the report in the same place. 

Thank you!

**Comment by dragonfly-lover** [comment]
I can confirm the same problem. I have a gtx1050ti 4gb with 415.23 drivers and Proton 3.16-5. previously i had the same problem also with a 750ti and 396.05 drivers.

**Comment by NoXPhasma** [comment]
I tried it over the last months with every Proton version and different drivers (Currently Proton 3.16-6 and nvidia 415.27) without any luck. The glitches are still present.

**Comment by NoXPhasma** [comment]
I've opened a topic on the Nvidia dev forums, you can add your experience there as well: https://devtalk.nvidia.com/default/topic/1051288/linux/artefacts-in-deadlight-when-running-with-proton-or-wine/

**Comment by RiantAnda** [comment]
I've tested out D9VK r0.1.3f with this game and the graphics glitches seem to be gone.
Proton hopefully picks up D9VK in the near future.

There is a thread over at D9VK for this game, seems like a workaround was needed:
https://github.com/Joshua-Ashton/d9vk/issues/124

**Comment by NoXPhasma** [comment]
I can confirm that installing latest D9VK build fixes the render glitches. Nice catch, @RiantAnda.

**Comment by NoXPhasma** [comment]
Also works like a charm with Proton 4.11-1 when using `PROTON_USE_D9VK=1 %command%`.

**Comment by LeJimster** [comment]
Unfortunately 4.11-1's d9vk build isn't new enough to fix rendering glitch AMD users experience, but it has been fixed since 0.13f, so it should be good for the next proton release.  The glitch nvidia users are experiencing must be different to https://github.com/Joshua-Ashton/d9vk/issues/124

**Comment by dragonfly-lover** [comment]
i can confirm that setting d9vk proton variable solves every glitch (at least in 4.11-2).
